docassemble/EFSPIntegration/efm_client.py

```diff
@@ -62,6 +62,8 @@ def user_visible_resp(resp):
          some issues in Docassemble if you are using this in interviews. This function
          takes the essentials of a response and puts in into a simple object.
       """
+      if resp is None:
+        return ApiResponse(200, 'Not yet implemented', None)
```

Comment:
Should probably have it as some other HTTP code besides 200, so it's clear it's not working. 501: Not Implemented is a good fit.